README.md

```diff
@@ -10,24 +10,75 @@
   ~ ----------------------------------------------------------------------------
   ~
 -->
+<p align="center">
+  &emsp;
+  <a href="https://deepchecks.com/blog/">Blog</a>
+  &emsp; | &emsp; 
+  <a href="https://docs.deepchecks.com">Documentation</a>
+  &emsp; | &emsp; 
+  <a href="https://join.slack.com/t/deepcheckscommunity/shared_invite/zt-y28sjt1v-PBT50S3uoyWui_Deg5L_jg">Join&nbsp;Slack</a>
+  &emsp; | &emsp;  
+  <a href="https://twitter.com/deepchecks">Twitter</a>
+  &emsp;
+</p>
+
+<p align="center">
+   <img src="docs/images/deepchecks-banner.png">
+</p>
+
+
 # Deepchecks - Test Suites for ML Models and Data
 
-![pyVersions](https://img.shields.io/pypi/pyversions/deepchecks)
-![pkgVersion](https://img.shields.io/pypi/v/deepchecks)
 ![build](https://github.com/deepchecks/deepchecks/actions/workflows/build.yml/badge.svg)
 [![Documentation Status](https://readthedocs.org/projects/deepchecks/badge/?version=latest)](https://docs.deepchecks.com/en/latest/?badge=latest)
+![pkgVersion](https://img.shields.io/pypi/v/deepchecks)
+![pyVersions](https://img.shields.io/pypi/pyversions/deepchecks)
+
+Validate your machine learning models and data, during the research and development phase. 
+
+With only one line of code you can discover data integrity problems, distribution mismatches,
+and efficiently evaluate your models to find potential vulnerabilities.
+
+## Installation
 
-Deepchecks is a Python package for comprehensively validating your machine learning
-models and data with minimal effort.
-This includes checks related to various types of issues, such as model performance,
-data integrity, distribution mismatches, and more.
+### Using pip
+```bash
+pip install deepchecks #--upgrade --user
+```
+### Using conda
+```bash
+conda install deepchecks
```

Comment:
@ItayGabbay does this need -c deepchecks as well?

Reply:
Yes, thanks.
```suggestion
conda install -c deepchecks deepchecks
```